examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<blockquote>for i in range(shape[0]):</blockquote><blockquote>&nbsp;&nbsp;&nbsp;v.data[0, i] = max((smin + ds0 * i) - K, 0)</blockquote><p>This can be done in one line right?
</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1288/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I added another operator for the example without a boundary condition which added a line to the for loop so I left it how it was